include/deal.II/base/parameter_handler.h

```diff
@@ -2334,7 +2334,7 @@ ParameterHandler::add_parameter(const std::string &          entry,
 
   declare_entry(entry,
                 Patterns::Tools::Convert<ParameterType>::to_string(
-                  parameter, pattern.clone()),
+                  parameter, *pattern.clone()),
```

Comment:
Here and elsewhere, you don't actually need the `clone()` call any more.
```suggestion
                  parameter, pattern),
```